src/main/java/com/powsybl/openloadflow/network/impl/LfLegBranch.java

```diff
@@ -194,17 +194,36 @@ public void updateFlows(double p1, double q1, double p2, double q2) {
                 .setQ(q1 * PerUnit.SB);
     }
 
-    public static ThreeWindingsTransformerResult createThreeWindingsTransformerResult(LfNetwork network, String threeWindingsTransformerId) {
+    private double getV() {
```

Comment:
You don't need these 2 methods, no?

Reply:
They are only there to factorize the null test.

Reply:
Use the methods of  `AbstractLfImpedantBranch`, no?

Reply:
````
            result.addExtension(OlfThreeWindingsTransformerResult.class, new OlfThreeWindingsTransformerResult(
                    leg1.getV1() * leg1.legRef.get().getTerminal().getVoltageLevel().getNominalV(),
                    leg2.getV1() * leg2.legRef.get().getTerminal().getVoltageLevel().getNominalV(),
                    leg3.getV1() * leg3.legRef.get().getTerminal().getVoltageLevel().getNominalV(),
                    Math.toDegrees(leg1.getAngle1()),
                    Math.toDegrees(leg2.getAngle1()),
                    Math.toDegrees(leg3.getAngle1())));
````